Lib/asyncio/events.py

```diff
@@ -763,3 +763,14 @@ def set_child_watcher(watcher):
     _c__set_running_loop = _set_running_loop
     _c_get_running_loop = get_running_loop
     _c_get_event_loop = get_event_loop
+
+
+#  shortcuts
+
+def create_task(coro):
+    """Schedule the execution of a coroutine object in a spawn task.
+
+    Return a Task object.
+    """
+    loop = get_running_loop()
+    return loop.create_task(coro)
```

Comment:
What if you pass a Task instance to this function? I'd say we check if coro is a coroutine here and raise an error if it's not.

Reply:
Isn't `loop.create_task()` perform the check?
Actually python implementation has it but C has not.

Reply:
Well, Python version has only an `assert` statement... Let's have an always working check in `tasks.py/Task` and `_asynciomodule.c/Task`.  It will add a bit of overhead, but will improve the usability.

`_asynciomodule.c/Task` can have a fast path using `PyCoro_CheckExact`.